### ISSUE TITLE
Build in Windows with Visual Studio 2019

### ISSUE THREAD

**Issue by madoodia** [PixarAnimationStudios/USD#1041]
Hello Guys
### Description of Issue
Boost does not build, because of adding toolset to `msvc-14.1` when VisualStudio is greater than 2017 (`here I'm using 2019`)
This will happen because msvc version on VS-2019 is `14.23...`, however I changed the toolset to `14.2` but didn't work and gave this error:
```
ERROR: Failed to run 'b2 --prefix="D:\USD\build" --build-dir="D:\USD\build\build" -j8 address-model=64 link=shared runtime-link=shared threading=multi variant=release --with-atomic --with-program_options --with-regex --with-python toolset=msvc-14.1 install'
See D:\USD\build\src\boost_1_65_1\log.txt for more details.
```
### Steps to Reproduce
make these lines commented
in `InstallBoost` function
```python
        # if Windows():
        #     if IsVisualStudio2017OrGreater():
        #         b2_settings.append("toolset=msvc-14.1")
        #     else:
        #         b2_settings.append("toolset=msvc-14.0")
```
After commenting these lines Boost is built successfully
### System Information (OS, Hardware)
Windows 10
Visual Studio 2019
Developer Command Prompt x64 (`vcvarsall` is run)
### Build Flags
```
$ python build_scripts\build_usd.py --generator "Visual Studio 16 2019" --python --alembic --hdf5 --houdini --houdini-location "D:/Program Files/Side Effects Software/Houdini 17.5.425" --maya --maya-location "D:/Program Files/Autodesk/Maya2018" build
```

**Comment by jilliene** [comment]
Filed as internal issue #USD-5710

**Comment by jcowles** [comment]
Hey @sunyab Was this fix included in 20.05 or is this still pending release?

**Comment by sunyab** [comment]
Hey @jcowles this fix is in the 20.05 release, just missed removing the tag.
